contracts/axelar-gateway/src/event.rs

```diff
@@ -25,9 +25,15 @@ pub(crate) fn approve_message(env: &Env, message: Message) {
         .publish(topics, (message.source_chain, message.source_address));
 }
 
-pub(crate) fn execute_contract_call(env: &Env, message_id: String) {
-    let topics = (symbol_short!("executed"), message_id);
-    env.events().publish(topics, ());
+pub(crate) fn execute_contract_call(env: &Env, message: Message) {
+    let topics = (
+        symbol_short!("executed"),
+        message.message_id,
```

Comment:
Shall we index the `source_chain`, `message_id`, and `payload_hash` for both approved/executed events instead in the topic? @sdaveas

Reply:
updated to have the full message data which would be more simpler for both approved/executed events.